src/components/ec-panel/ec-panel.spec.js

```diff
@@ -38,15 +38,38 @@ describe('EcPanel', () => {
       expect(wrapper.find('.ec-panel').exists()).toBe(false);
       expect(wrapper.element).toMatchSnapshot();
     });
+
+    it(':goBackEnabled - should render the back button within the panel when is true', () => {
+      const wrapper = mountPanel({ show: true, goBackEnabled: true });
```

Comment:
these props shoudl be named `isSomething`, ask @jaimeperezpedra for more context

Reply:
Amended.